.github/workflows/node.yml

```diff
@@ -177,3 +177,39 @@ jobs:
             - name: Test compatibility
               run: npm test -- -t "set and get flow works"
               working-directory: ./node
+
+
+    build-linux-musl-latest:
+        runs-on: ubuntu-latest
+        container:
+            image: node:18.19.0-alpine
+        timeout-minutes: 15
+        steps:
```

Comment:
This also needs to have `bash` and `gettext` tools
Please add:
`- name: Install bash
              run: |
                apk add bash
                apk add gettext`

Reply:
Same as noted [here](https://github.com/aws/glide-for-redis/pull/1267#discussion_r1564849763) , 
both `bash` and `gettext` are installed as part of `musl-dev`.
I tested it few times (can be tested on your own too :) ),  no additional packages are needed.
Of course we can explicitly specifying other packages as well like you and Yuri suggested,    but I really don't see the point.
WDYT?   Whatever you guys decide:)